packages/lodestar/src/chain/eventHandlers.ts

```diff
@@ -181,9 +184,11 @@ export async function onBlock(
 
   this.reprocessController.onBlockImported({slot: block.message.slot, root: blockRoot}, advancedSlot);
 
+  const delaySec = Date.now() / 1000 - (this.genesisTime + block.message.slot * this.config.SECONDS_PER_SLOT);
```

Comment:
Looks like this logic to calculate `delaySec` is already provided in `clock.secFromSlot(slot: Slot)`

Reply:
@tuyennhv Are you sure this metric won't be triggered during sync, and unknown block root fetches?

Reply:
I added BlockSource enum and track that in the new metric, please have a look